### DISCUSSION TITLE
iNav - Matek 765 board: I have no servo outputs. I ruled out the common setup errors. Please review my CLI output?

### DISCUSSION THREAD

**Discussion by shresthanator** [iNavFlight/inav#10188] [Q&A]
Hi there!

I need help getting my board to control the servos. I've checked for setup errors, ensured the output is turned on, and verified that the motor runs. However, I'm still waiting for a response from the servos. I get nothing. I played around with different mode setups, etc. 

I'm starting to suspect it could be a hardware issue, such as a faulty board or soldering problems.

As I've been out of the hobby for a while, I'm open to the possibility of making a setup error. I've included the CLI output for your review. I'm eager to learn and improve, so could you please help me figure this out?

[INAV_7.1.2_cli_20240701_105816.txt](https://github.com/user-attachments/files/16055206/INAV_7.1.2_cli_20240701_105816.txt)

**Comment by sensei-hacker** [comment]
First, on the Receiver tab of Configurator,.if you check one channel at a time, does each channel move correctly on the Receiver tab?

**Reply by shresthanator** [reply]
Hi Sensi, yup, the channels do move properly. I am using CRSF, so I had to select the correct type of TX. But each channel responds correctly (throttle is throttle, roll is roll, etc.) to my controller inputs. 

Also, when I unplug the board from the computer and arm it, I can move the throttle up and down- the motor responds like it should. I get absolutely nothing from the servos.

**Reply by shresthanator** [reply]
[latest cli - servos still do not work](https://github.com/user-attachments/files/16060408/INAV_7.1.1_cli_20240702_011225.txt)

**Reply by MrD-RC** [reply]
Have you checked that the servo outputs have power?
